manuscript/paper.md

```diff
@@ -33,16 +33,22 @@ Side-scan sonars are widely used instruments for high-resolution mapping of the
 
 Annotation of the targets is typically done using specialised software which does not produce results allowing for quick conversion to formats commonly used to train machine learning models.
 Our software `SideScanSonarEditor` is a free and open-source Python package which allows the user to read sonar data from XTF files, display it as images and allow further processing (XTF FILE FORMAT, n.d.).
-The key feature include the ability for: full analysis of the sonar imagery, simple manipulation of the sonar imagery and annotation of the objects of interest.
+The key features include the ability for: full analysis of the sonar imagery; simple manipulation of the sonar imagery; and annotation of the objects of interest.
```

Comment:
Alternatively, a bulleted list would work well instead of `;` to separate the listed features.